Source/fault.h

```diff
@@ -10,16 +10,16 @@ typedef struct STACK_FRAME {
 //int dword_52B9F4;
 extern LPTOP_LEVEL_EXCEPTION_FILTER lpTopLevelExceptionFilter; // idb
 
-void __cdecl exception_install_filter();
-void __cdecl j_exception_init_filter();
-void __cdecl exception_init_filter(void);
+void __cdecl fault_init_filter();
+void __cdecl fault_cleanup_filter_atexit();
+void __cdecl fault_cleanup_filter(void);
 LONG __stdcall TopLevelExceptionFilter(PEXCEPTION_POINTERS ExceptionInfo);

```

Comment:
May one know how had you decided on the new names: do you found them referenced somewhere or just made up?

Reply:
Essentially, inferred from functionality. And prefixing all names with `fault` (instead of `exception`), as it is likely that the file was named `fault.cpp`, given alphabetical linking order constraint and a maximum file name of 8 characters.

Reply:
Yeah, the name came from sanctuary originally which named it `exception.cpp`. That however exceeds the 8-character limit. It was renamed to `fault.cpp` after finding in Diablo 2's assertions a file exists with the same name and very similar code.

Reply:
So glad you have a good memory for these things @galaxyhaxz! Now the notes repo has been updated to mention this fact: https://github.com/sanctuary/notes/commit/5b94abab801ba85d8aaeccecde496037a73b73a7